docs/core-features/table_lookups.md

```diff
@@ -197,20 +197,20 @@ And you perform a Table Lookup using `4`. The result you should get is `lut[4] =
 
 The probability of this error can be configured through the `p_error` and `global_p_error` configuration options. The difference between these two options is that, `p_error` is for individual TLUs but `global_p_error` is for the whole circuit.
 
-If you set `p_error` to `0.01`, for example, it means every TLU in the circuit will have a 99% chance of being exact with a 1% probability of error. If you have a single TLU in the circuit, `global_p_error` would be 1% as well. But if you have 2 TLUs for example, `global_p_error` would be almost 2% (`1 - (0.99 * 0.99)`).
+If you set `p_error` to `0.01`, for example, it means every TLU in the circuit will have a 99% chance (or more) of being exact. If there is a single TLU in the circuit, it corresponds to `global_p_error = 0.01` as well. But if we have 2 TLUs, then `global_p_error` would be higher: that's `1 - (0.99 * 0.99) ~= 0.2`.
```

Comment:
`1 - (0.99 * 0.99)` is `~0.02` not `~0.2`

Reply:
And I'd keep 2%

Reply:
I should be ashamed! 

I've written `1 - (0.99 * 0.99) ~= 0.02 = 2%` to please everyone.